waku/node/v2/waku_types.nim

```diff
@@ -132,5 +132,9 @@ proc notify*(filters: Filters, msg: WakuMessage) =
         filter.handler(msg.payload)
 
 proc generateRequestId*(): string =
-  for _ in .. 10:
-    add(result, char(rand(int('A') .. int('z'))))
\ No newline at end of file
+  let 
+    rng = crypto.newRng()
```

Comment:
Should not be doing this for every `generateRequestId`. Doing it some where in an init, store this ref in an object and pass it as parameter in this call.